src/integration/pushResult.js

```diff
@@ -45,17 +45,21 @@ function addCommit(src, msg){
 //pulling from remote repo
```

Comment:
I think that we should not hide these logs. They are helpful, when some stuff is not appearing on production and the plan is green.